Sources/PostgreSQL/Connection/PostgreSQLConnection.swift

```diff
@@ -127,6 +128,24 @@ public final class PostgreSQLConnection: DatabaseConnection, BasicWorker {
         /// return the newly enqueued work's future result
         return new
     }
+    
+    /// Ask the server if it supports SSL and adds a new OpenSSLClientHandler to pipeline if it does
+    /// This will throw an error if the server does not support SSL
+    internal func addSSLClientHandler(using tlsConfiguration: TLSConfiguration) -> Future<Void> {
+        return queue.enqueue([.sslSupportRequest(PostgreSQLSSLSupportRequest())]) { message in
+            guard case .sslSupportResponse(let response) = message else {
+                throw PostgreSQLError(identifier: "SSL support check", reason: "Unsupported message encountered during SSL support check: \(message).", source: .capture())
+            }
+            guard response == .supported else {
+                throw PostgreSQLError(identifier: "SSL support check", reason: "tlsConfiguration given in PostgresSQLConfiguration, but SSL connection not supported by PostgreSQL server", source: .capture())
```

Comment:
Error messages should be grammatically correct (sentences should end with a period).